.github/workflows/ci.yaml

```diff
@@ -19,7 +19,34 @@ env:
   PR_PLATFORMS: linux/amd64,linux/arm64
 
 jobs:
+  detect-changes:
+    name: Detect Changes
+    runs-on: ubuntu-latest
+    outputs:
+      go_changed: ${{ steps.filter.outputs.go_changed }}
+      docker_changed: ${{ steps.filter.outputs.docker_changed }}
+      helm_changed: ${{ steps.filter.outputs.helm_changed }}
```

Comment:
there could be another output for `prerequisites` changed
- that `prerequisites.mk` changes often by renovate, we shouldn't skip it, it may break something
  - checking what is in it, it only affects some linting, helm-unit-test, and generated files (pls double check)
- in the future we could split it so it is more fine tuned

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/4849/commits/0f49d5d9b69bdba6fa8e2ec3730b01199883fc5f